lib/auth.guard.ts

```diff
@@ -64,6 +66,8 @@ function createAuthGuard(type?: string | string[]): Type<CanActivate> {
 
     handleRequest(err, user, info, context): TUser {
       if (err || !user) {
+        err && this.logger.error(err);
+        info && this.logger.error(info.toString());
```

Comment:
`info` is not a string as documentation on `passport` lib.  It's an Error object and `toString` is necessary.
As info probably is an error,  I use `logger.error` intentionally.
I tested with invalid `jwksUri` and invalid `issuer`.  What I get of `info` is Error type.
I said probably, I mean I am not sure it's true for all strategies.